lib/upipe-modules/upipe_udp_source.c

```diff
@@ -251,12 +294,66 @@ static void upipe_udpsrc_worker(struct upump *upump)
             upipe_throw_source_end(upipe);
         }
         return;
+    } else {
+        for(struct cmsghdr *cmsg = CMSG_FIRSTHDR(&msg); cmsg; cmsg = CMSG_NXTHDR(&msg,cmsg)) {
+            if( cmsg->cmsg_level != SOL_SOCKET) {
+                continue;
+            }
+
+            if (cmsg->cmsg_type != SCM_TIMESTAMPNS) {
```

Comment:
SCM_TIMESTAMPNS should be detected (it doesn't exist on OS X and perhaps other BSD flavours).

Reply:
This might go away